CITATION.cff

```diff
@@ -27,51 +30,57 @@ version: "0.8.2"
-  website: "https://mfisher87.github.io/"
-- family-names: "Kaufman"
-  given-names: "Daniel"
-  orcid: "https://orcid.org/0000-0002-1487-7298"
-  website: "https://github.com/danielfromearth"
-- family-names: "Kennedy"
-  given-names: "Joseph"
-  orcid: "https://orcid.org/0000-0002-9348-693X"
-  website: "https://github.com/jhkennedy"
-- family-names: "Lopez"
-  given-names: "Luis"
-  orcid: "https://orcid.org/0000-0003-4896-3263"
-  website: "https://github.com/betolink"
-- family-names: "Lowndes"
-  given-names: "Julia"
-  orcid: "https://orcid.org/0000-0003-1682-3872"
-  website: "https://github.com/jules32"
-- family-names: "Scheick"
-  given-names: "Jessica"
-  orcid: "https://orcid.org/0000-0002-3421-4459"
-  website: "https://github.com/JessicaS11"
-- family-names: "Steiker"
-  given-names: "Amy"
-  orcid: "https://orcid.org/0000-0002-3039-0260"
-  website: "https://github.com/asteiker"
+  - family-names: "Barrett"
+    given-names: "Andrew"
+    orcid: "https://orcid.org/0000-0003-4394-5445"
+    website: "https://github.com/andypbarrett"
+  - family-names: "Battisto"
+    given-names: "Chris"
+    orcid: "https://orcid.org/0000-0002-9608-3634"
+    website: "https://github.com/battistowx"
+  - family-names: "Bourbeau"
+    given-names: "James"
+    orcid: "https://orcid.org/0000-0003-2164-7789"
+    website: "https://github.com/jrbourbeau"
+  - family-names: "Fisher"
+    given-names: "Matt"
+    orcid: "https://orcid.org/0000-0003-3260-5445"
+    website: "https://mfisher87.github.io/"
+  - family-names: "Kaufman"
+    given-names: "Daniel"
+    orcid: "https://orcid.org/0000-0002-1487-7298"
+    website: "https://github.com/danielfromearth"
+  - family-names: "Kennedy"
+    given-names: "Joseph"
+    orcid: "https://orcid.org/0000-0002-9348-693X"
+    website: "https://github.com/jhkennedy"
+  - family-names: "Lopez"
+    given-names: "Luis"
+    orcid: "https://orcid.org/0000-0003-4896-3263"
+    website: "https://github.com/betolink"
+  - family-names: "Lowndes"
+    given-names: "Julia"
+    orcid: "https://orcid.org/0000-0003-1682-3872"
+    website: "https://github.com/jules32"
+  - family-names: "Scheick"
+    given-names: "Jessica"
+    orcid: "https://orcid.org/0000-0002-3421-4459"
+    website: "https://github.com/JessicaS11"
+  - family-names: "Steiker"
+    given-names: "Amy"
+    orcid: "https://orcid.org/0000-0002-3039-0260"
+    website: "https://github.com/asteiker"
 
 
 
 references:
-- type: "grant"
-  institution:
-    name: "National Aeronautics and Space Administration"
-  number: "20-TWSC20-2-0003"
+  - type: "grant"
+    title: "Openscapes: Supporting better science for future us"
+    institution:
+      name: "National Aeronautics and Space Administration"
+    number: "20-TWSC20-2-0003"
+    authors:
+      - family-names: "Lowndes"
+        given-names: "Julia"
+        orcid: "https://orcid.org/0000-0003-1682-3872"
+        website: "https://github.com/jules32"
```

Comment:
@jules32 `title` and `authors` are required fields now, and I took a shot, not my best shot, but a compromise between goodness/fastness :) Did I get the title/authors of the grant correct?

Reply:
Hi @MattF-NSIDC ! Awesome! Please also add: 

```
      - family-names: "Robinson"
        given-names: "Erin"
        orcid: "https://orcid.org/0000-0001-9998-0114"
        website: "https://erinrobinson.info/"
```

Reply:
:heavy_check_mark: d9994e1